.github/workflows/lint-test.yml

```diff
@@ -7,4 +7,76 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - uses: pantheon-systems/validate-readme-spacing@v1
\ No newline at end of file
+      - uses: pantheon-systems/validate-readme-spacing@v1
+  lint:
+    name: PHPCS Linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Cache dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/vendor
+          key: test-lint-dependencies-{{ checksum "composer.json" }}
+          restore-keys: test-lint-dependencies-{{ checksum "composer.json" }}
```

Comment:
I know this is copy-pasta so maybe a question for @jazzsequence, but should we be using composer-lock.json instead here? Seems that could change without composer.json changing, right?

Reply:
I would think `composer.lock` would change _more_ than `composer.json` since a `composer update` would bump the versions in the lockfile but not in the `composer.json` itself. At any rate, I don't think it's worth worrying about, this is what I've seen most frequently (and it's what CoPilot auto-completes to).

Reply:
If the goal is to cache vendor/, I believe we would want a change to composer-lock to nuke the cache, because otherwise vendor/ will be stale. But composer install after the fact should make up for the difference anyway and it's still a slimmer download to cover the diff.